prclz/etl/split_bldgs.py

```diff
@@ -0,0 +1,104 @@
+import argparse
+from typing import List, Union
+from logging import basicConfig, info
+from pathlib import Path 
+
+import geopandas as gpd
+import pandas as pd
+
+def gadm_dir_to_path(gadm_dir: str) -> str:
+	"""
+	For a given country,the GADM dir contains multiple file levels
+	so this convenience function just returns the path to the highest
+	resolution gadm file within the directory, which changes from 
+	country to country
+
+	Args:
+		gadm_dir: directory containing all gadm files for a country
+	
+	Returns:
+		Path to specific gadm file
+	"""
+	sort_fn = lambda p: int(p.stem.split("_")[-1])
+	gadm_dir = Path(gadm_dir)
+	files = [p for p in gadm_dir.iterdir() if ".shp" in p.name]
+	files.sort(key=sort_fn)
+	return files[-1]
```

Comment:
consider this one liner:
```python
return max(gadm_dir.glob("*.shp"), key = lambda p: int(p.stem.split("_")[-1]))
```

and if you choose to implement it this way, we can inline it at the call site

Reply:
I think it's nice having a little function to go from gadm dir to the highest res gadm file so I'm going to leave as is. But I'll move this to utils which seems like more of a natural home